raiden-dapp/src/locales/en.json

```diff
@@ -342,6 +347,11 @@
       "problem": "We do not know this error, sorry.",
       "solution": "Please file a new issue on https://github.com/raiden-network/light-client."
     },
+    "RDN_MINT_FAILED": {
+      "title": "Could not mint token",
+      "problem": "It looks like the Light Client does not support minting out-of-the-box for this token contract.",
+      "solution": "You may try to mint manually via MetaMask and Etherscan. You can find a step-by-step guide on https://github.com/raiden-network/light-client#minting-manually."
```

Comment:
Nitpick: couldn't this be converted in the link properly? Having the address as [non-clickable] text is kind of ugly

Reply:
No it can't otherwise I would have done it